### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by onyevyerov**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+                     " give 1 additional year for humans live"),
+    ]
+)
+def test_function_returns_good_result(cat_age: int, dog_age: int, human_age_to_animal: list):
+    assert get_human_age(cat_age, dog_age) == human_age_to_animal
+
+
+def test_age_pets_if_not_integer() -> None:
+    with pytest.raises(TypeError):
+        get_human_age("1", "1")
```
Comment (Dimosphen1) [inline app/test_main.py:33]: Use `@pytest.mark.parametrize` here as well. Test not only against strings, but against, dicts, lists, tuples, e.g. as well

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+                     id="not enough cat/dog years to 2 humans years"),
+        pytest.param(24, 24, [2, 2],
+                     id="first 24 dog/cat years gives 2 humans years"),
+        pytest.param(27, 27, [2, 2],
+                     id="not enough cat/dog years to 3 humans years"),
+        pytest.param(28, 28, [3, 2],
+                     id="28 years of dogs live is not enough for 3 humans years"),
+        pytest.param(100, 100, [21, 17],
+                     id="after first 24 years of dog/cat every 5 for dog and every 4 for cats lives"
+                     " give 1 additional year for humans live"),
```
Comment (Dimosphen1) [inline app/test_main.py:24]: Add test cases to cover negative input values and cases when `cat_age` and `dog_age` have different input values as mentioned in the checklist

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!
